auth/src/main/java/org/aerogear/auth/credentials/UsernamePasswordCredential.java

```diff
@@ -0,0 +1,19 @@
+package org.aerogear.auth.credentials;
+
+public class UsernamePasswordCredential implements ICredential {
```

Comment:
we are not going to support username/password credentials. So better delete this one.